CASSANDRA.md

```diff
@@ -16,7 +16,7 @@ All examples have been properly tested, and they should work out of the box.
 
 ## Prerequisites ##
 
-The keyspace and column family needed for all examples are defined in this CQL script:
+The keyspace and column family needed for all examples are defined in this CQL script. This step can be entirely skipped in Phantom were schema is auto-generated from the mapping DSL, and databases can be auto-generated using `database.create`.
```

Comment:
Please remove this text or move it to another section. This section is only here to define the schema, it's not part of the comparison.